components/common/WalletModal.vue

```diff
@@ -24,32 +25,38 @@
         </b-field>
       </section>
       <section v-if="hasUserWalletAuth" class="modal-card-body">
-        <div class="has-text-centered">
-          <img
-            src="~/assets/Koda_Beta.svg"
-            alt="First NFT market explorer on Kusama and Polkadot"
-            height="32" />
-        </div>
-
-        <div v-show="!hasSelectedWalletProvider" class="buttons my-5">
+        <div v-show="!hasSelectedWalletProvider" class="buttons">
           <b-button
             v-for="(wallet, index) in wallets"
             :key="index"
             size="is-medium"
             icon-right="chevron-right"
             expanded
             @click="setWallet(wallet)">
-            <b-image
-              :src="wallet.img"
-              class="is-24x24"
-              style="display: inline-block; vertical-align: middle"></b-image>
-            {{ wallet.name }}
+            <span class="pl-4">
+              <b-image
+                :src="wallet.img"
+                class="is-32x32"
+                style="display: inline-block; vertical-align: middle"></b-image>
```

Comment:
you can make a self-closing tag out of this and use dedicated bulma classes instead of styles
`is-inline-block`